package.json

```diff
@@ -235,6 +234,7 @@
   },
   "dependencies": {
     "macos-notification-state": "2.0.2",
+    "nan": "github:mvitale1989/nan#fix/electron-29",
```

Comment:
Proper way would be to install `nan@2.18.0` then patch using https://www.npmjs.com/package/patch-package so you don't need to maintain your own fork and eventually bump the version once available..

Reply:
Interesting, I didn't know that existed, thanks Saturn! I'll re-implement it using patch-package later today

Reply:
This is now implemented